src/stim/circuit/circuit_pybind_test.py

```diff
@@ -1345,3 +1345,32 @@ def test_num_ticks():
         CX 0 1
         TICK
     """).num_ticks == 2
+
+
+def test_reference_sample():
+    circuit = stim.Circuit(
+        """
+        H 0
+        CNOT 0 1
+    """
+    )
+    ref = circuit.reference_sample()
+    assert len(ref) == 0
+    circuit = stim.Circuit(
+        """
+        H 0
+        M 0
+        M 1
+    """
+    )
+    ref = circuit.reference_sample()
+    assert ref[0] == ref[1]
```

Comment:
I am hesitant to guarantee what the *specific* reference sample is, across versions. I think a better test might be...

```
H 0 1
CX 0 2 1 3
MPP X0*X1 Y0*Y1 Z0*Z1
```

Assert that there's an odd number of non-zero results.